docs/install.rst

```diff
@@ -281,33 +338,63 @@ you want to access, you will have to get the certificates from the corresponding
 information is available in their user guide.
 
 
+.. _troubleshooting:
+
 Troubleshooting
 =======================
 
 **Missing virtualenv**
 
-This should return the version of the RADICAL-Pilot installation, e.g., `0.X.Y`.
+This should return the version of the RADICAL-Pilot installation, e.g., `1.0.0`.
 
 If virtualenv **is not** installed on your system, you can try the following.
 
 .. code-block:: bash
 
-        wget --no-check-certificate https://pypi.python.org/packages/source/v/virtualenv/virtualenv-1.9.tar.gz
-        tar xzf virtualenv-1.9.tar.gz
+        wget --no-check-certificate http://pypi.python.org/packages/source/v/virtualenv/virtualenv-16.7.9.tar.gz
```

Comment:
I think this command is outdated. See `https://virtualenv.pypa.io/en/latest/installation/`. I get: 
```
$ wget --no-check-certificate http://pypi.python.org/packages/source/v/virtualenv/virtualenv-16.7.9.tar.gz
--2020-01-29 10:49:32--  http://pypi.python.org/packages/source/v/virtualenv/virtualenv-16.7.9.tar.gz
Resolving pypi.python.org (pypi.python.org)... 2a04:4e42:1b::223, 151.101.12.223
Connecting to pypi.python.org (pypi.python.org)|2a04:4e42:1b::223|:80... connected.
HTTP request sent, awaiting response... 403 SSL is required
2020-01-29 10:49:32 ERROR 403: SSL is required.
```